python/mxnet/symbol/symbol.py

```diff
@@ -2627,6 +2628,15 @@ def detach(self):
     def backward(self):
         raise NotImplementedForSymbol(self.backward, None)
 
+    def optimize_for_dynamic_shape_op(self):
+        """Check if any dynamic shape op presents in the symbol, if yes, partition all static shape ops for optimization
+        returns the optimized symbol.
+        """
+        out = SymbolHandle()
+        check_call(_LIB.MXOptimizeForDynamicShapeOp(self.handle, ctypes.byref(out)))
+        from .numpy import _Symbol as np_symbol
```

Comment:
you can import it once on top of file

Reply:
I tried to put it on the top. It causes import loop error